recipe/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  version: "1.8.0"
+
+package:
+  name: meson
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/m/meson/meson-${{ version }}.tar.gz
+  sha256: 0a9b23311271519bd03dca12d7d8b0eab582c3a2c5da433d465b6e519dc88e2f
+  patches:
+    # See conda-forge/glib-feedstock#40 for documentation regarding the RPATH patch
+    - patches/0001-Only-fix-RPATH-if-install_rpath-is-not-empty.patch
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install . -vv
+  noarch: python
+  python:
+    entry_points:
+      - meson = mesonbuild.mesonmain:main
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - ninja >=1.8.2
+    - setuptools
```

Comment:
Just noticed this, `setuptools` hasn't been a runtime dependency for quite a while, can we please remove this?

It came in 5 years ago for a usage of `pkg_resources`, but that's long gone.

Reply:
Done!